src/hidra/sender/eventdetectors/watchdog_events.py

```diff
@@ -489,7 +489,7 @@ def check_last_modified(self, filepath):
             return
 
         # compare ( >= limit)
-        if time_current - time_last_modified >= self.time_till_closed:
+        if (time_current - time_last_modified >= self.time_till_closed and os.path.getsize(filepath) > 0):
```

Comment:
For performance reasons, it would probably be better to reuse the return value of the earlier `stat` call instead of querying the filesystem a second time.

Please also add a comment to the code that describes the reason for the added check